### PR TITLE
[SU-4] build fastapi skeleton

### PR DESCRIPTION
**Pull request by jiajiapan**

### CODE REVIEW

**Review by shenzhu — COMMENTED** [review]
Can you add a README file on how to start the server?

**Comment by jiajiapan** [comment]
> Can you add a README file on how to start the server?

Sure, I have done.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+To start this server, you need to run this command on the terminal:
+uvicorn main:app --reload
```
Comment (shenzhu) [inline README.md]: let's use markdown to wrap the command

Comment (jiajiapan) [reply README.md]: OK

**Review by shenzhu — APPROVED** [review]

**Pull request merged**